MeetU/MeetU/Scripts/Services/MeetupDataService.js

```diff
@@ -0,0 +1,17 @@
+﻿(function () {
```

Comment:
this service is totally wrong. you should make a service to store information of currently logged in user, instead of just make a property.